addons/rook/template/base/install.sh

```diff
@@ -361,7 +361,21 @@ function verify_rook_updated_cluster() {
     fi
 
     log "Verifying Ceph version ${ceph_version} deployed"
-    if ! spinner_until 1200 rook_ceph_version_deployed "${ceph_version}" ; then
+    if ! "$DIR"/bin/kurl rook wait-for-ceph-version "${ceph_version}-0" --timeout=1200 ; then
```

Comment:
I'm curious why the argument to the command is `${cpeh_version}-0` with `-0` suffix.

Reply:
So far, it seems to work and that ceph version added is contacted with `-0`.
However, it might not work with future versions